.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: go
+
+go:
+  - "1.11"
+
+services:
+  - docker
+
+script: make clean validate test
```

Comment:
should we add here integration-test? or its already in the `test`

Reply:
good catch!